db/schema.rb

```diff
@@ -10,14 +10,15 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2022_05_13_100913) do
+ActiveRecord::Schema[7.0].define(version: 2022_05_17_110140) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
   create_table "eligibility_checks", force: :cascade do |t|
     t.boolean "free_of_sanctions"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.boolean "recognised"
```

Comment:
At some point we might want to track the `rejection_reason`. One for later, just to keep in mind.